### PR TITLE
Compute rolling update min/max limits

### LINKED ISSUE #206: Configure rolling update
ECS support rolling deployments policies. As `compose up` updates an existing application using CloudFormation changeset, those will be applied. Service must be created with rolling policy and parameters set according to `deploy` section in compose file (optionaly relying on custom extensions when relevant)

### PR DESCRIPTION
**Pull request by ndeloof**
**What I did**
Compute ECS DeploymentController min/max limits based on `parallelism`.
Also introduced aws extension for explicit control on those limits

**Related issue**
closes #206

**(not mandatory) A picture of a cute animal, if possible in relation with what you did**
image

### CODE REVIEW

File: pkg/amazon/backend/cloudformation.go [hunk 1]
```diff
@@ -232,6 +237,46 @@ func (b Backend) Convert(project *types.Project) (*cloudformation.Template, erro
...
+	}
+	updateConfig := service.Deploy.UpdateConfig
+	min, okMin := updateConfig.Extensions[compose.ExtensionMinPercent]
+	if okMin {
+		minPercent = min.(int)
+	}
+	max, okMax := updateConfig.Extensions[compose.ExtensionMaxPercent]
+	if okMax {
+		maxPercent = max.(int)
+	}
```
Comment (glours) [inline pkg/amazon/backend/cloudformation.go:254]:
```suggestion
	if max, okMax := updateConfig.Extensions[compose.ExtensionMaxPercent]; okMak {
		maxPercent = max.(int)
	}
```

File: pkg/amazon/backend/cloudformation.go [hunk 2]
```diff
@@ -232,6 +237,46 @@ func (b Backend) Convert(project *types.Project) (*cloudformation.Template, erro
...
+	maxPercent := 200
+	minPercent := 100
+	if service.Deploy == nil || service.Deploy.UpdateConfig == nil {
+		return minPercent, maxPercent, nil
+	}
+	updateConfig := service.Deploy.UpdateConfig
+	min, okMin := updateConfig.Extensions[compose.ExtensionMinPercent]
+	if okMin {
+		minPercent = min.(int)
+	}
```
Comment (glours) [inline pkg/amazon/backend/cloudformation.go:250]:
```suggestion
	if min, okMin := updateConfig.Extensions[compose.ExtensionMinPercent]; okMin {
		minPercent = min.(int)
	}
```

File: pkg/amazon/backend/cloudformation.go (hunk 1 above)
Comment (ndeloof) [reply pkg/amazon/backend/cloudformation.go:254]: nope, we need okMax variable later

Comment (glours) [reply pkg/amazon/backend/cloudformation.go:254]: Ok my bad 😉

**Pull request merged**